sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs

```diff
@@ -708,7 +708,7 @@ public virtual async Task<Response<CertificateWithPolicy>> ImportCertificateAsyn
         /// <param name="includePending">Specifies whether to include certificates in a pending state as well</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>An enumerable collection of certificate metadata</returns>
-        public virtual IEnumerable<Response<CertificateBase>> GetCertificates(bool? includePending = default, CancellationToken cancellationToken = default)
+        public virtual SyncCollection<CertificateBase> GetCertificates(bool? includePending = default, CancellationToken cancellationToken = default)
```

Comment:
How are devs supposed to get the raw response? I thhought that was one of the draws of thin clients. It's been a discussion point about `CryptographyClient`, for example, whether to expose the underlying remote crypto client so devs could access the raw response.

Reply:
They can use ByPage to get raw response, it's also a represents the underlying logic better because we are not getting a response per item - it's a response per page.

This doesn't change the pattern for methods that do not return lists.